torch_glow/src/PyTorchModelLoader.h

```diff
@@ -279,6 +279,13 @@ class PyTorchModelLoader {
   /// Rescale a int8 NodeValue \p input to the equivalent uint8 NodeValue.
   glow::NodeValue rescaleIntToUint(glow::NodeValue input);
 
+  /// Load a quantized conv node from ptNode to qconv.
+  /// a wrapper function of loadQuantizedConv and loadQuantizedConvRelu.
+  /// Returns error on failure.
+  Expected<NodeValue>
+  getQuantizedConvTransposeNode(const torch::jit::Node *ptNode,
```

Comment:
can we name this some thing like `loadQuantizedConvImpl` so it's clear what its doing